packages/web-react/src/components/Tooltip/TooltipAnchor.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { ElementType } from 'react';
+import { StyleProps } from '../../types';
+import { useTooltipContext } from './TooltipContext';
+
+interface TooltipAnchorProps extends StyleProps {
+  elementType?: ElementType | string;
+  children: string | React.ReactNode | ((props: { isOpen: boolean }) => React.ReactNode);
+}
+
+const TooltipAnchor = (props: TooltipAnchorProps) => {
```

Comment:
I am not quite sure about this naming myself. The "anchor" name collides, for me, with the anchor element (`a`, e.g. the link). I have asked the AI for suggestions and it came with the `TooltipTarget`. What do you think about this name?

Reply:
This was just test file when i was trying figure outm external tooltip, i will remove this file